src/qt/forms/createwalletdialog.ui

```diff
@@ -17,6 +17,48 @@
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QLabel" name="label_description">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
+     </property>
+     <property name="text">
+      <string>You are one step away from creating your new descriptor wallet!</string>
```

Comment:
This line just seems extra clutter. In any case please don't use the scary word "descriptor".

Reply:
Agree that there's no reason to mention "descriptor" here.

Reply:
Yeah sure. Word removed 👍🏼 .

About the extra line topic:
Based on my experience, the GUI should be explicative on what is going to happen. An actionable pop up with no description and only buttons is quite bad in terms of UX.

Still, this line is just a tiny tiny thing. I think this dialog is just bad. Would be nice to re-write it as a wizard and improve its design in the future.